eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md

```diff
@@ -26,31 +26,27 @@ Running this sample will be charged by Azure. You can check the usage and bill a
 ### Create Azure resources
 
 1.  Create [Azure Event Hubs][create-event-hubs].
-    Please note `Basic` tier is unsupported. After creating the Azure Event Hub, you
+    After creating the Azure Event Hubs, you
     can create your own Consumer Group or use the default "$Default" Consumer Group.
 
 1.  Create [Azure Storage][create-azure-storage] for checkpoint use.
 
-1.  **[Optional]** if you want to use service principal, please follow
-    [create service principal from Azure CLI][create-sp-using-azure-cli] to create one.
-
-1.  **[Optional]** if you want to use managed identity, please follow
-    [create managed identity][create-managed-identity] to set up managed identity.
-
```

Comment:
We can retain this part and udpate the application-sp/application-mi configuration files.

Reply:
I'd update this after verified

Reply:
updated